x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_logs_step/verify_changes/verify_changes.tsx

```diff
@@ -0,0 +1,131 @@
+    <FormattedMessage
+      id="xpack.upgradeAssistant.overview.verifyChanges.calloutTitle"
+      defaultMessage="{warningsCount, plural, =0 {No} other {{warningsCount}}} deprecation {warningsCount, plural, one {warning} other {warnings}} since {previousCheck}"
+      values={{
+        warningsCount,
+        previousCheck: (
+          <>
+            <FormattedDate value={previousCheck} year="numeric" month="long" day="2-digit" />{' '}
+            <FormattedTime value={previousCheck} timeZoneName="short" hour12={false} />
+          </>
+        ),
+      }}
+    />
+  ),
+  calloutBody: i18n.translate('xpack.upgradeAssistant.overview.verifyChanges.calloutBody', {
+    defaultMessage:
+      'Reset the counter after making changes and continue monitoring to verify that you are no longer using deprecated APIs.',
+  }),
+  loadingError: i18n.translate('xpack.upgradeAssistant.overview.verifyChanges.loadingError', {
+    defaultMessage: 'An error occurred while retrieving the count of deprecation logs',
+  }),
+  retryButton: i18n.translate('xpack.upgradeAssistant.overview.verifyChanges.retryButton', {
+    defaultMessage: 'Try again',
+  }),
+  resetCounterButton: i18n.translate(
+    'xpack.upgradeAssistant.overview.verifyChanges.resetCounterButton',
+    {
+      defaultMessage: 'Reset counter',
+    }
+  ),
+};
+
+const getPreviousCheckpointDate = () => {
+  const storedValue = moment(localStorage.get(LS_SETTING_ID));
+
+  if (storedValue.isValid()) {
+    return storedValue.toISOString();
+  }
+
+  const now = moment().toISOString();
+  localStorage.set(LS_SETTING_ID, now);
+
+  return now;
+};
+
+export const VerifyChanges: FunctionComponent = () => {
+  const { api } = useAppContext();
+  const [previousCheck, setPreviousCheck] = useState(getPreviousCheckpointDate());
+  const { data, error, isLoading, resendRequest } = api.getDeprecationLogsCount(previousCheck);
+
+  const warningsCount = data?.count || 0;
+  const calloutTint = warningsCount > 0 ? 'warning' : 'success';
+  const calloutIcon = warningsCount > 0 ? 'alert' : 'check';
+  const calloutTestId = warningsCount > 0 ? 'hasWarningsCallout' : 'noWarningsCallout';
+
+  useInterval(() => {
+    resendRequest();
+  }, POLLING_INTERVAL);
+
+  const onResetClick = () => {
+    const now = moment().toISOString();
+
+    setPreviousCheck(now);
+    localStorage.set(LS_SETTING_ID, now);
+  };
+
+  if (!data && isLoading) {
+    return (
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiLoadingContent lines={6} />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  }
+
+  if (error) {
+    return (
+      <EuiCallOut
+        title={i18nTexts.loadingError}
+        color="danger"
+        iconType="alert"
+        data-test-subj="errorCallout"
+      >
+        <p>
+          {error.statusCode} - {error.message}
+        </p>
+        <EuiButton color="danger" onClick={resendRequest} data-test-subj="errorResetButton">
```

Comment:
[nit] Maybe rename the `data-test-subj` to `retryButton` or something like that?